README.md

```diff
@@ -3,12 +3,50 @@ Cyface Android SDK
 
 This project contains the Cyface Android SDK which is used by Cyface applications to capture data on Android devices.
 
-- [How to integrate the SDK](#how-to-integrate-the-sdk)
-- [Migration from Earlier Versions](#migration-from-earlier-versions)
+- [Integration Guide](#integration-guide)
+- [API Usage Guide](#api-usage-guide)
+- [Migration Guide](#migration-guide)
+- [Developer Guide](#developer-guide)
```

Comment:
I added sections which explain how to get the library from the Registry and how to publish a new version. However, I'll configure the CI, too, so that new commits on the master trigger a new publish automatically.